asreview/webapp/src/HomeComponents/DashboardComponents/Projects.js

```diff
@@ -90,7 +90,7 @@ const ProjectCard = ({
   const [deleteDialog, toggleDeleteDialog] = useToggle();
   const [openSetup, toggleSetup] = useToggle();
 
-  const review = project["reviews"][0];
+  const review = project?.reviews?.[0];
```

Comment:
Can you explain why these changes are needed? Having a project without reviews should be impossible by design.

Reply:
I saw your comment on Slack. Is there a problematic project in your `~/.asreview` folder? Maybe empty the folder to check. I will look into this a bit.